workers/unity/Packages/com.improbable.gdk.core/Worker/ConnectionHandlers/CommandMetaData.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using System.Collections.Generic;
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core
+{
+    public struct CommandContext<T>
+    {
+        public Entity SendingEntity;
+        public T Request;
+        public object Context;
+
+        public CommandContext(Entity sendingEntity, T request, object context)
+        {
+            SendingEntity = sendingEntity;
+            Request = request;
+            Context = context;
+        }
+    }
+
+    public class CommandMetaData
```

Comment:
This name is somewhat confusing as this stores _all_ command meta data rather than being a particular instance of command meta data.